Tests/FunctionalTests/RunFunctionalTests.js

```diff
@@ -99,7 +100,11 @@ function main() {
         if (path === "-h" || path === "--help") {
             return help();
         }
-    } else {
+        if (process.argv.indexOf("-perfmode") !== -1) {
```

Comment:
Why is `--help` in `path` and `-perfMode` in `process.argv`?

Reply:
Path is always the first argument, and help can only be the first argument, just a reuse of the variable